libs/platform/ebpf_epoch.c

```diff
@@ -220,22 +234,20 @@ ebpf_epoch_initiate()
     ebpf_assert(EBPF_CACHE_ALIGN_POINTER(_ebpf_epoch_cpu_table) == _ebpf_epoch_cpu_table);
 
     for (cpu_id = 0; cpu_id < _ebpf_epoch_cpu_count; cpu_id++) {
```

Comment:
Will this become a performance bottleneck if the underlying hardware has a high core count? For example, [this product range](https://www.supermicro.com/en/products/mp) from supermicro can have up to 480 cores.

Reply:
Resolved per discussion with Alan